packages/react-flex/src/components/Flex/useFlexStyles.ts

```diff
@@ -2,10 +2,19 @@ import { mergeClasses, makeStyles } from '@fluentui/react-make-styles';
 import { FlexState } from './Flex.types';
 
 const useStyles = makeStyles({
-  // TODO: Add default styles
-  root: {},
-  // TODO: Add styles conditioned on FlexState props, for example:
-  // TODO: Add styles for any other slots, for example:
+  root: {
+    '--gap': 0,
```

Comment:
**suggestions:** can we move the var tokens so it's referenced only once ? currently they are used 3 times.

```ts
const cssVars = {
  gap: '--fui-flex-gap',
  grow: '--fui-flex-grow',
  shrink: '--fui-flex-shrink,
}


const useStyles = makeStyles({
   [cssVars.gap]: 0,
   [cssVars.grow]: 0,
  ....,
'> *': {
      order: 0,
      margin: `var(${cssVars.gap})`,   // NOTE: NO IDEA IF OUR BUILD TIME COMPILER PROCESSES LITERAL STRING EVALUATION
    ....
})
```